source/installation/install-kafka-cluster.txt

```diff
@@ -0,0 +1,146 @@
+.. _install-kafka-cluster:
+
+======================================
+Install with an Existing Kafka Cluster
+======================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator on an existing Kafka cluster including 
+Kafka Connect. This installation method is ideal if you are already running 
+Kafka or you have your own approach to installing and configuring Kafka.
+
+Before you Begin 
+----------------
+
+You must have Kafka and a Kafka Connect cluster accessible to the computer 
+running Relational Migrator.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install Debezium Source Connect Plugin
+
+      1. Download the source connector archive for your source database.
```

Comment:
**Nit**

For substeps, I have been using letters instead of numbers, would you consider changing this throughout for consistency?

```suggestion
      a. Download the source connector archive for your source database.
```

Reply:
updated.